pkg/lightning/backend/backend.go

```diff
@@ -89,6 +89,9 @@ type EngineConfig struct {
 	// KeepSortDir indicates whether to keep the temporary sort directory
 	// when opening the engine, instead of removing it.
 	KeepSortDir bool
+	// TS is the preset timestamp of data in the engine. When it's 0, the used TS
+	// will be set lazily.
```

Comment:
```suggestion
	// will be set lazily.
	// only used in global sort now
```

Reply:
it's also used in `generateLocalEngineConfig` in local fast reorg of DDL